### ISSUE TITLE
static act purge IP adresses prevent successful purge

### ISSUE THREAD

**Issue by thet** [eea/eea.docker.varnish#18]
A backends.vcl file is generated with an static acl purge list - see: https://github.com/eea/eea.docker.varnish/blob/master/varnish/src/add_backends.py#L65

At least in my setup (nginx->eea varnish->eea haproxy->plone) plone cannot successfully purge content when it's outdated.

Instead of the hardcoded 172.17.0.0/16 and 10.42.0.0/16 a config option or broader netmaskj would be fine. In my case one of the Plone backends have a IP of ``172.21.0.4``.

**Comment by thet** [comment]
This is my whole generated backends.vcl:

```
backend server_loadbalancer_0 {
    .host = "loadbalancer";
    .port = "80";
    .probe = {
        .url = "/";
        .timeout = 1s;
        .interval = 1s;
        .window = 3;
        .threshold = 2;
    }
}

import std;
import directors;
import saintmode;
sub vcl_init {

  new cluster_loadbalancer = directors.round_robin();

  cluster_loadbalancer.add_backend(server_loadbalancer_0);
}
acl purge {
    "localhost";
    "172.17.0.0/16";
    "10.42.0.0/16";
}

sub vcl_recv {
  if (req.method == "PURGE") {
    if (!client.ip ~ purge) {
        return(synth(405, "Not allowed."));
    }
    return (purge);
  }

  set req.backend_hint = cluster_loadbalancer.backend();
}


```

**Comment by avoinea** [comment]
@thet You're right. I'll add an environment variable for it.

**Comment by vdwijngaert** [comment]
Would be a nice improvement :)

**Comment by valentinab25** [comment]
solved in 4.1-6.5
